bevy_script_api/src/rhai/bevy/mod.rs

```diff
@@ -138,15 +131,19 @@ impl CustomType for ScriptWorld {
                     })
                 },
             )
-            .with_fn("get_children", |self_: &ScriptWorld, parent: Entity| {
+            .with_fn("get_parent", |self_: ScriptWorld, entity: Entity| {
+                if let Some(parent) = self_.get_parent(entity) {
```

Comment:
How does this work on the rhai side? Do you have to check the type of the result every time?

We also have support for options similarly to Vec with the custom wrapper, the only thing is it needs to be registered for the specific type with  `app.register_foreign_rhai_type::<Option<T>>()` I believe.

I agree though it is a bit annoying having to register all these, so assuming that returning another type is ergonomic this may be the way to go at least with the std lib for bevy. Can we use `()` instead though ?

Reply:
Alright, I tried it out again, I'm not sure what I did different now, but I now return a UNIT (`()`) and it works in Rhai.
I've expanded the multiple_events example to check this.
in Rhai it would look like this:
```rhai
let parent = world.get_parent(entity);
switch type_of(parent) {
    "bevy_ecs::entity::Entity" => {
        print(`Parent is an Entity`);
    },
    "()" => {
        print(`Parent doesn't exist`);
    },
}
```

Reply:
> How does this work on the rhai side? Do you have to check the type of the result every time?

Not necessarily. 
If you're not 100% if the parent exists, then this is recommended yes.
But if you're sure you can just use the value returned from `get_parent`.